README.md

```diff
@@ -1,40 +1,95 @@
-# eslint-config-semistandard
+# eslint-config-semistandard [![travis][travis-image]][travis-url] [![npm][npm-image]][npm-url] [![downloads][downloads-image]][downloads-url] [![javascript style guide][semistandard-image]][standard-url]
 
-[![npm][npm-image]][npm-url]
-[![travis][travis-image]][travis-url]
-
-[npm-image]: https://img.shields.io/npm/v/eslint-config-semistandard.svg?style=flat-square
-[npm-url]: https://www.npmjs.com/package/eslint-config-semistandard
-[travis-image]: https://img.shields.io/travis/standard/eslint-config-semistandard.svg?style=flat-square
+[travis-image]: https://img.shields.io/travis/standard/eslint-config-semistandard/master.svg
 [travis-url]: https://travis-ci.org/standard/eslint-config-semistandard
+[npm-image]: https://img.shields.io/npm/v/eslint-config-semistandard.svg
+[npm-url]: https://npmjs.org/package/eslint-config-semistandard
+[downloads-image]: https://img.shields.io/npm/dm/eslint-config-semistandard.svg
+[downloads-url]: https://npmjs.org/package/eslint-config-semistandard
+[semistandard-badge]: https://raw.githubusercontent.com/standard/semistandard/master/badge.svg
+[semistandard-image]: https://img.shields.io/badge/code_style-semistandard-brightgreen.svg
+[semistandard-url]: https://github.com/standard/semistandard
+[standard-url]: https://standardjs.com
+
+#### An ESLint [Shareable Config](http://eslint.org/docs/developer-guide/shareable-configs) for [JavaScript Semistandard Style][semistandard-url]
 
-eslint sharable config for semistandard – like [eslint-config-standard](https://github.com/feross/eslint-config-standard), but with semicolons
+[![JavaScript Style Guide - Standard Style][semistandard-badge]][semistandard-url]
 
 ## Install
 
-Use [`install-peerdeps`](https://npm.im/install-peerdeps):
+This module is for advanced users. You probably want to use [`semistandard`][semistandard-url] instead :)
 
+```bash
+npm install eslint-config-semistandard
 ```
-npx install-peerdeps --dev eslint-config-semistandard
+
+## Usage
```

Comment:
This whole section is a bloody mess.

At the start, a user is pointed towards official ESLint docs. If they really follow that link and learn how to use shareable configs, the rest of the section becomes virtually useless for them.

What follows is a semi-automatic/totally weird back-n-forth way of generating ESLint config plus installing this shareable config together with its dependencies. That is some sort of easy mode I guess.

If you wanna go hard and setup manually you are presented with a monstrous onliner that is somehow a clear win over using `install-peerdeps`.

And finally, following logical sequence, after easy and hard comes _releief_ mode suggesting to drop the ball and just use `semistandard` which is a completely different world compared to manual ESLint fine tunning...

Reply:
Agreed that it is messy, just reverted it. I had just copy-pasted the raw README from [eslint-config-standard](/standard/eslint-config-standard#readme) and s/standard/semistandard for consistency's sake.